### PR TITLE
[Pro splash pages] /features

### PR DESCRIPTION
**Pull request by anothertompetty**
Getting this in here now for early review. Content to come in a later PR

Just applies to `features.en.js`

image
(Ignore sticky nav duplication in scr shot)

### CODE REVIEW

**Comment by clippy** [comment]
**I have manually checked the visuals in:**

- [ ] Chrome
- [ ] IE11
- [ ] Firefox
- [ ] Safari
- [ ] iPad

**Comment by anothertompetty** [comment]
@jamesshedden you okay to take a look?

File: app/css/components/page-hero.scss [hunk 1]
```diff
@@ -34,6 +34,13 @@
   background-repeat: no-repeat;
 }
 
+.page-hero--features {
+  background: $primary-color url('/images/hero/hero-icons.jpg');
+  background-size: cover;
+  background-position: 50% 50%;
+  background-repeat: no-repeat;
+}
+
```
Comment (jamesshedden) [inline app/css/components/page-hero.scss]: As this is just the same rules as `.page-hero--holding-page`, let's just re-use `.page-hero--holding-page` where we need it and maybe rename to `.page-hero--icons-background`

File: app/pages/features/features.en.js [hunk 2]
```diff
@@ -45,20 +37,19 @@ export default class FeaturesEn extends React.Component {
             </div>
           </Translation>
           <Translation locales='en-GB'>
-            <div className='page-hero u-relative u-size-full'>
+            <div className='page-hero page-hero--medium page-hero--features u-relative u-size-full'>
```
Comment (jamesshedden) [inline app/pages/features/features.en.js]: @anothertompetty As mentioned above we can re-use existing class for this

File: app/pages/features/features.en.js [hunk 3]
```diff
@@ -107,408 +98,223 @@ export default class FeaturesEn extends React.Component {
...
-                  <h3 className='u-text-xs'>Export your data</h3>
-                  <p className='u-size-5of6 u-color-dark-gray u-margin-Bm'>
-                    View your data the way you want. GoCardless is compatible with the most popular accounting packages.
+                  <h3 className='u-text-xs u-color-dark-gray'>Get notified when something happens</h3>
+                  <p className='u-color-dark-gray u-margin-Bm'>
+                    Provide a great user experience by adding your logo to our<br />payment forms and
+                    automatically sending customers back<br />to your site after paying.
                   </p>
-                  <h3 className='u-text-xs'>Submit payments every day</h3>
+                  <h3 className='u-text-xs u-color-dark-gray'>Ssearch for your custom data</h3>
```
Comment (jamesshedden) [inline app/pages/features/features.en.js]: `Ssearch` -> `Search`

**Comment by jamesshedden** [comment]
@anothertompetty Awesome stuff! Some quick comments. One other thing - this is probably a bit annoying to change, but it'd be nice to use JPGs instead of PNGs - not sure we need the transparency and would mean we can have smaller file sizes. If it's not too much hassle that'd be sweet :v:

**Comment by anothertompetty** [comment]
Thanks @jamesshedden.

Images will be changed later, so I'll make sure they're all jpgs later down the line :+1:

**Pull request merged**